home/templates/home/questions/new_question.html

```diff
@@ -0,0 +1,28 @@
+{% extends "home/base.html" %}
+{% load static %}
+{% block content %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body class="landing-bg">
```

Comment:
Since you extend "home/base.html" you don't need to write the html, head, and body tags
they all written already in base.html.
so the first tag should be div class=newfrm
Also for adding a title look at the comment I wrote on views.py

Reply:
I agree with you about the html and head, but as soon as I delete the body the background color we set will also be deleted. Do you have a way for the background color (defined by the "landing-bg" class) to remain while I delete the body?
Also, logically, I add the feature of adding the question to the body of the page. Do you still think it's right to delete the body completely from new_question.html?

Reply:
add  class="main-bg" to the body tag in base.html and it will take care of the background-color
the feature of adding the question will still be in the body of the page because you extend the base.html
where the " {% block content %}{% endblock %} " lays inside a body tag.

Reply:
I saw that there is no 'body' tag in base.html.
As far as I know, you should update this file now and also later.
I thought about deleting the body tag from new_question.html and after you finish updating the base.html the updates will be done automatically in new_question.html.
What do you think?

Reply:
sounds good

Reply:
Aviv, I was wrong, my apologies. you should add a body tag in your HTML and give it a background color.
In my new PR #71 "Fix html.base" I removed the color from the body, so this CSS won't apply to all pages and everyone will have control over the background color in their feature.
Moreover, writing a body tag inside your template - which extends a template that already has a body tad, won't make a corrupt structure or anything like that. it will merge the body tags into one - at least as I saw during my work.

Reply:
Ok, changed